### ISSUE TITLE
ItemFakeCopy

### ISSUE THREAD

**Issue by NicosaurusRex99** [rwtema/extrautilities_old_issues#1763]
https://pastebin.com/w46xz9cr

**Comment by SirTwsted** [comment]
Yup I can confirm this. When I back port to version 1.6.4 of Extra Utilities 2
as instructed here -->> https://github.com/rwtema/extrautilities/issues/1762
I also get a ItemFakeCopy crash along with a Throwables 'NullPointerException' Error.
.
https://pastebin.com/4QYwy46M
